js/runtime.js

```diff
@@ -572,16 +566,40 @@
         },
 
         shape: {
-            fillShape: function(shp){
-                shp();
+            fill: function(pathSet){
```

Comment:
There should be *some* way to create a shape without drawing it so we can grab a shape for masking, or for flowing text along (for instance). The default and easiest path can be to draw the shape, as long as there is a way to just get the shape to use elsewhere without drawing it.